### PR TITLE
Fix flavors and sizes

### PR DESCRIPTION
**Pull request by Usiel-Gandalf**
Fix flavor and sizes

### CODE REVIEW

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt [hunk 1]
```diff
@@ -40,36 +40,35 @@ import com.iteneum.designsystem.utils.getFileName
...
-    modifier: Modifier = Modifier,
+    modifier: Modifier,
     textButton: String,
     icon: ImageVector? = null,
     onClick: () -> Unit,
 ) {
     OutlinedButton(
         onClick = onClick,
-        modifier = modifier.padding(10.dp),
+        modifier.padding(10.dp),
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt]: Add name params (modifier and padding name param)

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt [hunk 2]
```diff
@@ -40,36 +40,35 @@ import com.iteneum.designsystem.utils.getFileName
...
         shape = RoundedCornerShape(12.dp),
-        border = BorderStroke(1.dp, MaterialTheme.colorScheme.primary),
+        border = BorderStroke(1.dp, MaterialTheme.colorScheme.onPrimary),
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt]: Add name params.

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt [hunk 3]
```diff
@@ -40,36 +40,35 @@ import com.iteneum.designsystem.utils.getFileName
...
-                        contentDescription = textButton,
-                        modifier = Modifier.size(ButtonDefaults.IconSize)
-                    )
-                    Spacer(Modifier.size(ButtonDefaults.IconSpacing))
-                }
-                Text(textButton)
+        Row(modifier = Modifier.fillMaxWidth()) {
+            icon?.let {
+                Icon(
+                    icon,
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt]: Add name param.

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt [hunk 4]
```diff
@@ -138,7 +137,7 @@ fun LpEditFloatingActionButton(
 fun BadgeButton(
     modifier: Modifier = Modifier,
     onClick: () -> Unit,
-    badgeNumber: Int
+    badgeNumber: Int,
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt:139]: Delete ","

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt [hunk 5]
```diff
@@ -40,36 +40,35 @@ import com.iteneum.designsystem.utils.getFileName
...
+        Row(modifier = Modifier.fillMaxWidth()) {
+            icon?.let {
+                Icon(
+                    icon,
+                    contentDescription = textButton,
+                    modifier = Modifier.size(ButtonDefaults.IconSize)
+                )
+                Spacer(Modifier.size(ButtonDefaults.IconSpacing))
             }
+            Text(textButton)
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt]: Add name param.

**Review by carlosph816 — COMMENTED** [review]
Waiting changes.

**Comment by Usiel-Gandalf** [comment]
Added params name

**Review by carlosph816 — APPROVED** [review]
Well done!

**Comment by melissaquezada94** [comment]
Please add the description of the PR

**Review by melissaquezada94 — APPROVED** [review]

**Comment by Usiel-Gandalf** [comment]
In this fix, i just change name flavors, it change nothing l, because how the palet flavors it change in the project, it showns other color.

**Pull request merged**